app/policies/portfolio_item_policy.rb

```diff
@@ -19,6 +19,8 @@ def destroy?
     update_portfolio_check
   end
 
+  alias restore? destroy?
```

Comment:
Feel confused in the beginning, prefer to define its own method, just like other permission checks.

Reply:
@hsong-rh We already have quite a few `alias` methods in various policies: [in service_plan_policy](https://github.com/RedHatInsights/catalog-api/blob/master/app/policies/service_plan_policy.rb#L10), [in application_policy on line 21](https://github.com/RedHatInsights/catalog-api/blob/master/app/policies/application_policy.rb#L21) and [line 27](https://github.com/RedHatInsights/catalog-api/blob/master/app/policies/application_policy.rb#L27), as well as [portfolio_policy on line 6](https://github.com/RedHatInsights/catalog-api/blob/master/app/policies/portfolio_policy.rb#L6) and [line 24](https://github.com/RedHatInsights/catalog-api/blob/master/app/policies/portfolio_policy.rb#L24).

Normally I would probably agree that we could just extract the logic into a private method and then have separate methods call that private method, but since most of these checks are just single lines, it doesn't really make sense and an alias is cleaner (though as you mentioned, maybe a bit confusing at first). Since the other files have them already though, I think it makes sense to keep it.

Reply:
👍